src/app/external-cluster-wizard/steps/summary/external-cluster-details/provider/aks-cluster-summary/template.html

```diff
@@ -0,0 +1,82 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<km-external-cluster-details [cluster]="cluster">
+  <ng-container leftColumn>
+    <km-property>
+      <div key>Location</div>
+      <div value>
+        <span>{{ spec.location }}</span>
+      </div>
+    </km-property>
+
+    <km-property>
+      <div key>Resource Group</div>
+      <div value>
+        <span>{{ aks.resourceGroup }}</span>
+      </div>
+    </km-property>
+  </ng-container>
+
+  <ng-container rightColumn>
+    <div class="section-header">Node Pool Setting</div>
+    <km-property>
+      <div key>Node Pool Name</div>
+      <div value>
+        <span>{{ spec.machineDeploymentSpec.name }}</span>
+      </div>
+    </km-property>
+
+    <ng-container *ngIf="spec.machineDeploymentSpec.basicSettings as setting">
+      <km-property>
+        <div key>VM Size</div>
+        <div value>
+          <span>{{ setting.vmSize }}</span>
+        </div>
+      </km-property>
+
+      <km-property>
+        <div key>Count</div>
+        <div value>
+          <span>{{ setting.count }}</span>
+        </div>
+      </km-property>
+
+      <km-property>
+        <div key>Mode</div>
+        <div value>
+          <span>{{ setting.mode }}</span>
+        </div>
+      </km-property>
+
+      <km-property-boolean label="AutoScaling"
+                           [value]="setting.enableAutoScaling"> </km-property-boolean>
+
+      <km-property>
+        <div key>Max Count</div>
+        <div value>
+          <span>{{ setting.scalingConfig.maxCount }}</span>
+        </div>
+      </km-property>
+
+      <km-property>
+        <div key>Min Count</div>
+        <div value>
+          <span>{{ setting.scalingConfig.minCount }}</span>
+        </div>
+      </km-property>
```

Comment:
i think here we need to check if autoscaling is enabled in order to show the max count and min count because right now there is a bug when in autoscaling is disabled 
![Screenshot from 2022-09-07 14-07-44](https://user-images.githubusercontent.com/85109141/188864021-4316f48f-7e86-4cbe-8bb7-deb2ea2f0cd1.png)